docs/start/install-qiskit-source.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Install Qiskit from source
+description: Install Qiskit from source
+
+---
+# Install Qiskit from source
+
+Installing Qiskit from source allows you to access the current development version, instead of using the version in the Python Package Index (PyPI) repository. This lets you inspect and extend the latest version of the Qiskit code more efficiently. 
+
+## Create and activate a new virtual environment
+
+First, create a new virtual environment for Qiskit. 
+
+```python
+  python3 -m venv QiskitDevenv
+  source QiskitDevenv/bin/activate
+```
+
+## Install the Rust compiler
+
+The Rust compiler must be installed in your new virtual environment. To install the Rust compiler, use the cross-platform Rust installer [rustup](https://rustup.rs/) or [another install method.](https://forge.rust-lang.org/infra/other-installation-methods.html) 
+
+## Install Qiskit
+
+Follow these steps to install Qiskit:
+
+1. Clone the Qiskit repository.
+
+```python
+  git clone https://github.com/Qiskit/qiskit-terra.git
+```
+
+2. Change to the `qiskit-terra` directory. 
+
+```python
+  cd qiskit-terra
+```
```

Comment:
````suggestion
```python
  git clone https://github.com/Qiskit/qiskit.git
```

2. Change to the `qiskit` directory. 

```python
  cd qiskit
```
````

the repository is called `Qiskit/qiskit` now